matrix-synapse-riot/terraform/matrix-server/debian-50unattended-upgrades

```diff
@@ -0,0 +1,105 @@
+// Unattended-Upgrade::Origins-Pattern controls which packages are
```

Comment:
Curious where these files come from?

Reply:
That is to enable auto apt update / upgrade on Debian.